bundles/nim/init.moon

```diff
@@ -0,0 +1,33 @@
+mode_reg =
+  name: 'nim'
+  extensions: 'nim'
+  create: bundle_load('nim_mode')
+
+howl.mode.register mode_reg
+
+howl.command.register
+  name: 'nim-run'
+  description: 'Compile and run the current Nim file'
+  handler: ->
+    exe = howl.config.nim_executable
+    unless exe
+      error 'Cannot locate nim executable, please define config.nim_executable.'
+    unless howl.io.File(exe).exists
+      error "Nonexistent path '#{exe}' - invalid config.nim_executable."
+    unless howl.app.editor.buffer.file
+      error "No file associated with current buffer."
+    howl.command.run "exec #{exe} c -r " .. howl.app.editor.buffer.file.basename
+
+unload = -> howl.mode.unregister 'nim'
+
+howl.config.define
+  name: 'nim_executable',
```

Comment:
Using 'nim' as the default is a good idea - it should just work if it's in the path. We don't need to know the full path.

Re nimsuggest, currently I've copied it in the directory containing nim, but I believe when fixed it will install automatically via nimble and may not be in the same directory. So that might need its own variable - not sure what we'd call that - 'nimble_executable' or 'nim_nimble_executable' (in case we want to namespace everything by the bundle name.